### PR TITLE
Don't require npm to build from source

### PR DESCRIPTION
**Pull request by stsewd**
We already have this scripts in package.json,
and we commit the static files in the repo.

Close https://github.com/readthedocs/sphinx_rtd_theme/issues/1037

### CODE REVIEW

**Comment by adamjstewart** [comment]
This also closes #1014

**Comment by stsewd** [comment]
Looks like warnings are shown when running `setup.py install|build`, but when running `pip install`, they are only shown with `pip install -v`.

**Comment by agjohnson** [comment]
I removed closing on #1014, yhis will not close #1014 as we'll want to come back to that issue to resolve the underlying bug with our source builds. For now we're only putting a hack in place.

File: setup.py [hunk 1]
```diff
@@ -1,27 +1,29 @@
 # -*- coding: utf-8 -*-
-"""`sphinx_rtd_theme` lives on `Github`_.
-
-.. _github: https://github.com/readthedocs/sphinx_rtd_theme
-
-"""
 
+import distutils.cmd
```
Comment (Blendify) [inline setup.py:3]: Picky: I prefer this below os and subprocess

**Review by Blendify — APPROVED** [review]
Nice, to show that this works, you can try searching the docs with the PR build and Latest

File: setup.py (hunk 1 above)
Comment (stsewd) [reply setup.py:3]: I used isort for this

Comment (Blendify) [reply setup.py:3]: In that case, don't worry about it, I don't have a strong preference.

**Comment by agjohnson** [comment]
For clarification, what we have decided to do here is to communicate deprecation of installation via Git for now, and separate the build step. It seems like the issue in #1037 is that projects are installing through Git -- an installation channel we don't want to support. We don't want to support this as rebuilding assets makes PRs conflict, and we're tied to making release decisions that also include people install unpinned versions of our theme.

For now, the build assets step will be moved, but this PR will be partially reverted later. We'll move back to removing static assets checked into this repository, and all builds should go through an official release.

**Review by agjohnson — APPROVED** [review]
This is a temporary fix, we need to next figure out deprecation plans.

**Pull request merged**